### ISSUE TITLE
Added tokens do not show up

### ISSUE THREAD

**Issue by BeardPower** [LedgerHQ/ledger-live-desktop#2592]
#### Ledger Live Version and Operating System

- tested on Ledger Live **1.20.0*
- Platform and version: **Windows 10**

#### Expected behavior
Show newly added ERC20-Tokens.

#### Actual behavior
After adding an ERC20-Token, it does not show up.

#### Steps to reproduce the behavior
Click on Accounts and add a new token by clicking on "Add token" and follow the steps.
The token does not show up; neither does empty token accounts.

**Comment by dacoinvid** [comment]
Seems like the same issue as me.. :/

I created a second Ethereum account recently, and I have send tokens to them.
Ledger Live Mobile & Desktop do not detect them. Whereas they are well visible on https://etherscan.io/tokenholdings?a=xxxxxxxx

I tried to send a bit of Ethereum to this account, both Ledger Live & Desktop show this Ethereum amount but always 0 tokens.. Remove & Adding account doesn't fix this issue.
